alerts.tf

```diff
@@ -73,4 +73,24 @@ module "div-bulkcase-errors-alert" {
   trigger_threshold_operator = "GreaterThan"
   trigger_threshold = 0
   resourcegroup_name = "${azurerm_resource_group.rg.name}"
+}
+
+module "div-data-extraction-alert" {
+  source = "git@github.com:hmcts/cnp-module-metric-alert"
+  location = "${var.location}"
+
+  app_insights_name = "div-${var.env}"
+
+  alert_name = "Data extraction to Family Man has not happened yet"
+  alert_desc = "Logs aren't indicating that e-mails with data extraction were sent in div-${var.env}."
+  app_insights_query = "traces | where message startswith 'Sent extracted data to' and tostring(customDimensions['LoggerName']) has 'dataextraction'"
+  custom_email_subject = "Alert: Data extraction does not seem to be working in div-${var.env}"
+  frequency_in_minutes = 120
+  time_window_in_minutes = 1440
+  severity_level = "2"
+  action_group_name = "div-support"
+  trigger_threshold_operator = "LessThan"
+  trigger_threshold = 3
+  resourcegroup_name = "${azurerm_resource_group.rg.name}"
+  enabled = "${var.env == "prod"}"
```

Comment:
We probably want this in AAT, since it used the same server maybe?

Reply:
If it's running everywhere, it will create alerts for the environments that don't have the scheduler enabled, like Demo, perf-tests, etc.

Reply:
`"${var.env == "prod" || var.env == "aat"}"` maybe

Reply:
@hosainnet, I'm thinking we should only apply for prod. The reason is that, if the "or" logic doesn't work, we'll only know it when the alert fails to notice the error.

Reply:
@jvictorcf we use the || logic in quite a few TF files
https://github.com/hmcts/div-case-maintenance-service/blob/master/infrastructure/main.tf#L7
I don't know, personally I'll try to avoid prod-only anything if possible